cspice/src/vector.rs

```diff
@@ -7,7 +7,7 @@ use derive_more::{Deref, DerefMut, From, Into};
 
 /// A 3D vector
 #[derive(Copy, Clone, Debug, Default, PartialEq, From, Into, Deref, DerefMut)]
-pub struct Vector3D([SpiceDouble; 3]);
+pub struct Vector3D(pub [SpiceDouble; 3]);
```

Comment:
it may also be a good idea to have a `impl From<Rectangular> for Vector3D` implementation